include/onnxruntime/core/common/common.h

```diff
@@ -282,6 +282,7 @@ inline std::string ToMBString(const std::string& s) { return s; }
  */
 std::string ToMBString(const std::wstring& s);
 
+std::string ToUTF8String(const std::wstring& s);
```

Comment:
Would you please help add some comments to the four functions?
It was already a mess. If you do ToMBString(ToWideString(..))) or  ToWideString(ToMBString(..))) , the result is a garbage. Here we need to make it more clear which encoding the multibyte strings are using.  You are making a good example here: The result of ToUTF8String is obviously UTF-8. But how about the others? And what if someone put the result ToUTF8String to ToWideString?